packages/wdio-browserstack-service/src/service.js

```diff
@@ -6,14 +6,14 @@ import { BROWSER_DESCRIPTION } from './constants'
 const log = logger('@wdio/browserstack-service')
 
 export default class BrowserstackService {
-    constructor (config) {
+    constructor (options, caps, config) {
```

Comment:
The argument `caps` is not being used, do we still need it in the constructor?

Reply:
same as above